googletest/src/gtest.cc

```diff
@@ -868,6 +872,10 @@ TimeInMillis GetTimeInMillis() {
   struct timeval now;
   gettimeofday(&now, NULL);
   return static_cast<TimeInMillis>(now.tv_sec) * 1000 + now.tv_usec / 1000;
+#elif GTEST_HAS_CHRONO_
+	std::chrono::time_point<std::chrono::system_clock> now = std::chrono::system_clock::now();
+	auto duration = now.time_since_epoch();
```

Comment:
time_since_epoch is not guaranteed to return a duration in milliseconds which is what this function is to return.  So one needs to cast it to what is expected.  

  const std::chrono::milliseconds nowMs{ std::chrono::duration_cast<std::chrono::milliseconds>(now) };
  return static_cast<TimeInMillis>(nowMs.count());